src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts

```diff
@@ -195,6 +195,10 @@ function getAdaptedState(state: PartialState<NavigationState<RootStackParamList>
             if (matchingRootRoute?.name === NAVIGATORS.FULL_SCREEN_NAVIGATOR) {
                 routes.push(createCentralPaneNavigator({name: SCREENS.SETTINGS.WORKSPACES}));
             }
+            if (matchingRootRoute?.state?.routes[0]?.name === SCREENS.SEARCH.CENTRAL_PANE) {
+                const centralPaneRoute = matchingRootRoute?.state?.routes[0];
```

Comment:
Maybe the logic responsible for excluding params for this screen should be in the `getMatchingCentralPaneRouteForState`? Instead of removing params we would simply not include them in the first place